resume-buddy/src/main/webapp/scripts/resume-review-script.js

```diff
@@ -7,6 +7,9 @@ function onLoad() {
  * Fetches comments from the servers and adds them to the DOM.
  */
 function getComments() {
+  /* TODO: default hide comments section
+  /* Only if the user has a match, display comments functionality
+   */
```

Comment:
Please make a corresponding GitHub Issue for this TODO and link to it in this comment.